packages/plugin-dev/src/browser/hosted-plugin-manager-client.ts

```diff
@@ -306,8 +323,53 @@ export class HostedPluginManagerClient {
         }
     }
 
-    protected getErrorMessage(error: Error): string {
-        return error.message.substring(error.message.indexOf(':') + 1);
+    protected async launchVSCode({ arguments: { args } }: LaunchVSCodeRequest): Promise<LaunchVSCodeResult> {
+        let result = {};
+        let instanceURI;
+
+        const sessions = this.debugSessionManager.sessions.filter(session => session.id !== this.connection.sessionId);
+
+        if (sessions.length > 0) {
+            sessions.forEach(session => session.terminate(false));
```

Comment:
add some comments about why you terminate existing sessions

Reply:
Added